gradle-module/src/main/java/org/example/Main.java

```diff
@@ -22,4 +23,11 @@ public static void main(String[] args) {
     private static void test() {
         System.out.println("test");
     }
-}
\ No newline at end of file
+
+   /* asd
+    we
+    asd
+
+    */
+    
+}
```

Comment:
There is no newline at the end of the file. It's a good practice to end files with a newline to avoid issues with certain tools and to adhere to POSIX standards. Please add a newline at the end of the file.